.travis.yml

```diff
@@ -50,6 +50,7 @@ script:
   - yarn tx
   - yarn test
   - yarn build:$COZY_APP_SLUG:browser
+  - ./scripts/deploy-tests
```

Comment:
No need to `chmod a+x` ?  :o

Reply:
I did `chmod` on the deploy-tests file locally, but it seems to run without it on travis

cozy-bank doesn't seem to do `chmod` either https://github.com/cozy/cozy-banks/blob/master/.travis.yml#L43

Reply:
Yeah if the file is marked as executable before pushing it it works on travis too 👍